app/controllers/api/v1/organizations_controller.rb

```diff
@@ -2,6 +2,8 @@ module Api
   module V1
 
     class OrganizationsController < ApplicationController
+      @@to_include = [:mailing_address, :practice_location_address, :other_provider_identifiers,
```

Comment:
Small thing, but in ruby it would be more typical to see a class constant (ie SERIALIZATION_INCLUDES) vs a class variable (particularly since it's not intended to vary at run time).